src/sinks/types.go

```diff
@@ -0,0 +1,12 @@
+package sinks
+
+import (
+	"github.com/cybertec-postgresql/pgwatch3/metrics"
+)
+
+type MeasurementMessage metrics.MeasurementMessage
+
+type WriteRequest struct {
```

Comment:
Meh. I don't like this. We have `metrics.MeasurementMessage.CustomTags` for that. Every sink writer can update it with whatever it needs to, e.g.

```go
msg.CustomTags['pgwatchID'] = 2
```